### PR TITLE
Describe auth in Prow

### PR DESCRIPTION
**Pull request by strekm**
**Description**

**Related issue(s)**

### CODE REVIEW

File: docs/prow/authorization.md [hunk 1]
```diff
@@ -2,10 +2,45 @@
 
 ## Required GCP Permissions
 
-Every Prow clsuter should be deployed in its own GCP project. You need an owner access to deploy Prow and configure it properly.
+Every Prow cluster should be deployed in its own GCP project. You need an owner access to deploy Prow and configure it properly. In addition you need following service accounts created:
```
Comment (jakkab) [inline docs/prow/authorization.md]:
```suggestion
In order to deploy a Prow cluster, configure the following service accounts in the GCP project you own. 
```

Comment (jakkab) [inline docs/prow/authorization.md]:
```suggestion
In order to deploy a Prow cluster, configure the following service accounts in the GCP project you own.
```

File: docs/prow/authorization.md [hunk 2]
```diff
@@ -2,10 +2,45 @@
...
+
+- sa-gcs-plank - service account used by Prow plan microservice with the `Storage Object Admin` role
```
Comment (jakkab) [inline docs/prow/authorization.md]:
```suggestion
| Service account name | description | required roles |
|:--------------------:|:-----------:|:--------------:|
| sa-gcs-plank | used by Prow plan microservice | Storage Object Admin |
```

Comment (jakkab) [reply docs/prow/authorization.md]: Please enclose the SA data in a chart like this

File: docs/prow/authorization.md [hunk 3]
```diff
@@ -2,10 +2,45 @@
...
+    - `configmaps`
+    - `secrets`
+    - `events`
+    - `services`
+    - `pods`
+    - `ingresses.extensions`
+
+The `cluster-admin` kubernetes role is granted to `Tiller` service account.  
+
+Following roles exist on Prow cluster:
```
Comment (jakkab) [inline docs/prow/authorization.md:31]: Please create a chart as well...

File: docs/prow/authorization.md (hunk 2 above)
Comment (strekm) [reply docs/prow/authorization.md]: done

File: docs/prow/authorization.md (hunk 3 above)
Comment (strekm) [reply docs/prow/authorization.md:31]: done

File: docs/prow/authorization.md (hunk 1 above)
Comment (strekm) [reply docs/prow/authorization.md]: done

**Pull request merged**